app/src/main/java/com/gdg/pune/devfest19/viewholder/SpeakerViewHolder.kt

```diff
@@ -29,6 +31,22 @@ class SpeakerViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         } else {
             profileView.setImageResource(R.drawable.ic_account_circle)
         }
+
+        val onClickListener = View.OnClickListener {
```

Comment:
As I said earlier, intent should be instantiated in adapter class for document id purpose.